### ISSUE TITLE
Check that archive node deb package works as expected

### ISSUE THREAD

**Issue by aneesharaines** [MinaProtocol/mina#5565]
The only change that should be required here is that the archive node deb should list postgresql as a dependency.

**Comment by robinbb** [comment]
@lk86 I wonder if @kaozenn can take this from you?

**Comment by mzajda0mina** [comment]
relates to #11976 specific for nix deb package

**Comment by robinbb** [comment]
@mzajda0mina Why is this issue marked "Archived in project"? Is it done?

**Comment by mzajda0mina** [comment]
@robinbb is a duplicate of #11976

**Comment by mzajda0mina** [comment]
@robinbb hm, yes, I was not correct re the duplicate ticket number. I did it during standup, where a duplicate was pointed to me, but now I don't see it... I will re-check. I see also that I added the [nix-dep] prefix in the title. It means that it depends on NIX work. If we get NIX working the issue doesn't exist. The title is misleading, too. It's about PostgreSQL dependency and not  Archive node working state.
 
Re the archiving non-closed issue: it was my way to remove it from this dashboard and don't lose it forever. When I have better grip on the project I intend to move or delete non issues.
